style/content.rst

```diff
@@ -0,0 +1,143 @@
+===============
+ Content Guide
+===============
+
+Principles
+==========
+
+* Use your judgement about following these recommendations
+  
+  * OK to consider ROI
+
+* Strive for consistency across components
+
+  * Especially newer specs
+  * Component must be consistent with itself
+
+* Target audience
+
+  * implementors
+  * sophisticated users who will be providing feedback
+
+    * assume final product will have separate programming guide with
+      tutorial-level information
+
+* Spec must be publishable as part of unified doc and as standalone
+  describing a single component
+
+  * when published as a standalone, it will contain some sections that
+    are normally at top-level. e.g. bibliography
+  
+Structure of Document
+=====================
+
+* Top-level spec contains sections for information common to all
```

Comment:
I would prefer a more traditional way to word it: 'should contain', etc. The rest of the document lacks verbs in many places. @andreynv : can you please edit the document to make it more readable? For example I'm not sure I understand what the 'Conventions' bullet really lists.

Reply:
As mentioned on the meeting, my doc - quick summary/theses on the spec structure considerations to discuss on the bridge, w/o any goal to have 'shall do'. The published version - reformatted one that can deviate from mine. It makes sense first to have thoughts of @rscohn2 on the next steps.

Reply:
The problem is that having missed the meeting I don't really understand how to read this. For example, what does 'Basic scalar data types' mean under the 'Conventions' ? Does it mean that my spec needs such section?

Reply:
The idea - have 'xIPL' conventions in the single place, on the top level. The components would have a short note on compliance with the conventions, alternatively, the note only in case of specific deviations